app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,18 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+
+    def __init__(self, nickname: str,
+                 musical_instrument: str,
+                 bow_level: int
+                 ) -> None:

```

Comment:
```

    def __init__(
             self,
             nickname: str,
             musical_instrument: str,
             bow_level: int
     ) -> None:
```

Reply:
There is an error in your proposed solution Flake8 "Closing bracket does not match visual indentation"

Reply:
There is an error in your proposed solution Flake8 "Closing bracket does not match visual indentation"